helper/DockerHelper.go

```diff
@@ -107,21 +107,37 @@ func (impl *DockerHelperImpl) StartDockerDaemon(commonWorkflowRequest *CommonWor
 			util.LogStage("Insecure Registry")
 		} else {
 			if connection == util.SECUREWITHCERT {
-				os.MkdirAll(fmt.Sprintf("/etc/docker/certs.d/%s", host), os.ModePerm)
-				f, err := os.Create(fmt.Sprintf("/etc/docker/certs.d/%s/ca.crt", host))
+				util.LogStage("Secure with Cert")
 
-				if err != nil {
+				// Create /etc/docker/certs.d/<host>/ca.crt with specified content
+				log.Printf("creating /etc/docker/certs.d/%s/ca.crt", host)
```

Comment:
log `certDir` constant defined in next line, instead formatting inside the log, otherwise both might evolve independently, and create inconstancy